apps/x509.c

```diff
@@ -1053,17 +1054,20 @@ int x509_main(int argc, char **argv)
             i = PEM_write_bio_X509(out, x);
     } else {
         BIO_printf(bio_err, "Bad output format specified for outfile\n");
-        goto end;
+        goto err;
     }
     if (!i) {
         BIO_printf(bio_err, "Unable to write certificate\n");
-        goto end;
+        goto err;
```

Comment:
Please, don't rename the label.

Reply:
It is not a rename her (Also see https://github.com/openssl/openssl/pull/17022)

Reply:
Sorry, I don't get. Why can't we use the old label and to print errors in case of non-empty error stack? 2 labels are badly maintainable, I think.

Reply:
I see your point, but I didn't want to make a big refactoring there.  The goal of that PR was to fix the wrong exit code and it did. A code refactoring should be done as a separate PR from my point of view.
Please if you have something related to that PR, put your comments there. This PR is related  to the s_client new flag. Thanks.